doc/nrf/known_issues.rst

```diff
@@ -1667,22 +1712,22 @@ nrfxlib
 Crypto
 ======
 
-.. rst-class:: v2-0-2 v2-0-1 v2-0-0
+.. rst-class:: v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

Comment:
v2-1-0 can be removed for NCSDK-15697 since it no longer applies

Reply:
Updated.